More precise wording. (#682)
SUSE/doc-susemanager: modules/installation/pages/install-server-unified.adoc

+

image::unifiedinstaller-addon.png[scaledwidth=80%]

> . OPTIONAL: In the [guimenu]``Add On Product`` screen, select any additional or add-on products you require, and click btn:[Next]. Note that it is not recommended to run any other workloads on your {productname}. So only add add-ons that you absolutely require, like driver repositories from your hardware vendor.

Reviewer: ```suggestion
. OPTIONAL: In the [guimenu]``Add On Product`` screen, select any additional or add-on products you require, and click btn:[Next]. 
We do not recommend that you run any other workloads on {productname}. 
Only use add-ons that you absolutely require, such as driver repositories from your hardware vendor.
```
